### ISSUE TITLE
Does this library have cache support?

### ISSUE THREAD

**Issue by uzaysan** [pankaj89/MasterExoPlayer#1]
Does this library cache vides and olay cached videos arter first olay or download video every time?

İf it doesnt have cache support how can i add it?

**Comment by pankaj89** [comment]
You can support caching of video url using this library
https://github.com/danikula/AndroidVideoCache

Instead of directly sending your video url to exoplayerhelper, send proxy url which comes from above library.

**Comment by uzaysan** [comment]
İm using this library but its old and doesnt work as expected on New android versions and doesnt work properly when connection is slow. 

Do you planning to add cache support to this library? İ know exoplayer has cache feature itself but im font have the information end experience to implement it bu myself?

**Comment by pankaj89** [comment]
Yes that is in my plan list and also implemented some of the stuff, but need to test and refactor based on new feature.

**Comment by uzaysan** [comment]
Ok im waiting for it.

**Comment by MogiLogi** [comment]
Hello, I tried to implement this 
https://proandroiddev.com/video-preloading-precaching-using-exoplayer2-in-android-da340cc268d9 
maybe it can help you.
